css/settings/_uswds-theme-components.scss

```diff
@@ -34,7 +34,7 @@ $theme-alert-padding-x: 2.5;
 //$theme-banner-background-color: "ink" !default;
 //$theme-banner-font-family: "lang" !default;
 //$theme-banner-link-color: default !default;
-//$theme-banner-max-width: "none" !default;
+$theme-banner-max-width: "widescreen";
```

Comment:
Setting this to `widescreen` helps our site display, but does not affect component previews